frontend/src/context/auth-context.tsx

```diff
@@ -69,19 +72,40 @@ function AuthProvider({ children }: React.PropsWithChildren) {
     }
   };
 
+  const logout = () => {
+    clearGitHubToken();
+    posthog.reset();
+  };
+
+  const refreshToken = async (): Promise<boolean> => {
+    const config = await OpenHands.getConfig();
+
+    if (config.APP_MODE !== "saas" || !gitHubTokenState) {
+      return false;
+    }
+
+    const newToken = await OpenHands.refreshToken(
+      gitHubTokenState,
+      config.APP_MODE,
+    );
```

Comment:
```suggestion
    const newToken = await OpenHands.refreshToken(config.APP_MODE);
```
You can apply this change if making the suggested change to to `refreshToken`